Sources/DistributedActors/Pattern/WorkerPool.swift

```diff
@@ -160,7 +160,7 @@ public distributed actor WorkerPool<Worker: DistributedWorker>: DistributedWorke
         return selected
     }
 
-    private func onWorkerTerminated(id: Worker.ID) {
+    public distributed func terminated(actor id: Worker.ID) {
```

Comment:
This new API works out rather nicely I think (e.g., even less code duplication)).